indico/modules/events/notifications.py

```diff
@@ -38,3 +40,18 @@ def notify_event_creation(event, occurrences=None):
     if emails:
         template = get_template_module('events/emails/event_creation.txt', event=event, occurrences=occurrences)
         send_email(make_email(bcc_list=emails, template=template))
+
+
+def notify_move_request(move_requests, accept, reason=''):
+    """Send email notifications when a move request is accepted/rejected.
+
+    :param move_requests: List of `EventMoveRequest` that were accepted/rejected.
+    :param accept: Whether the requests were accepted.
+    :param reason: Optional reason for rejection.
+    """
+
+    for email, requests in itertools.groupby(move_requests, lambda rq: rq.requestor.email):
```

Comment:
group by the user itself, not their email

Reply:
also, you need to make sure the list is sorted for this to work. you can use our `group_list` util that combines sorting and grouping